codingup/style.css

```diff
@@ -0,0 +1,321 @@
+    padding: 0 2.5rem;
+    line-height: 3.75rem;
+    display: inline-block;
+}
+
+.tour_categories {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 1.875rem 1.875rem;
+}
+
+.tour_categories > div {
+    position: relative;
+}
+
+.rome {
+    grid-column: 1 / span 2;
+}
+
+.tour_overlay {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: flex-start;
+    padding: 1.875rem;
+}
+
+a.button_two {
+    box-shadow: 0px 15px 30px rgba(0, 0, 0, 0.3);
+    border-radius: 5px;
+    background: var(--highlight_color);
+    color: var(--background_color_one);
+    font-weight: 500;
+    font-size: 0.875rem;
+    padding: 0.625rem;
+    margin-bottom: 1rem;
+    display: inline-block;
+}
+
+.tour_text {
+    font-size: 1.25rem;
+    font-weight: 700;
+    color: var(--background_color_one);
+}
+
+.best_offers {
+    min-height: 20rem;
+    background-color: var(--background_color_two);
+}
+
+.best_offers_title {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+}
+
+.best_offer_row {
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    padding: 1.25rem;
+    margin-left: 1.25rem;
+    margin-right: 1.25rem;
+    align-items: center;
+    font-size: 0.875rem;
+    color: var(--text_color_four);
+}
+
+.best_offer_row strong {
+    color: var(--text_color_two);
+    font-size: 1rem;
+}
+
+.best_offer_row img {
+    margin-left: -2.5rem;
+}
+
+.best_offer_row:nth-child(even) {
+    box-shadow: 0px 20px 40px rgba(0, 0, 0, 0.1);
+    border-radius: 5px;
+    background-color: var(--background_color_one);
+}
+
+.best_offer_row:nth-child(even) img {
+    box-shadow: 0px 20px 40px rgba(0, 0, 0, 0.3);
+}
+
+.blog {
+    text-align: center;
+}
+
+.blog_posts {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-column-gap: 0.625rem;
+}
+
+.blog_posts > div {
+    margin-top: 1.25rem;
+}
+
+.blog_posts img {
+    display: block;
+    margin: -1.25rem auto 0 auto;
+}
+
+.blog_posts > div:nth-child(even) {
+    box-shadow: 0px 20px 40px rgba(0, 0, 0, 0.1);
+    border-radius: 5px;
+}
+
+.blog_posts > div:nth-child(even) img {
+    box-shadow: 0px 30px 60px rgba(0, 0, 0, 0.3);
+}
+
+.blog_post_teaser {
+    margin: 2.5rem;
+}
+
+.blog_posts h1 {
+    font-size: 1.25rem;
+    font-weight: 700;
+    margin-top: 1.875rem;
+    margin-bottom: 0.625rem;
+    color: var(--text_color_two);
+    text-align: left;
+}
+
+.blog_posts p {
+    color: var(--text_color_four);
+}
+
+.blog_posts a {
+    color: var(--text_color_five);
+    font-weight: 500;
+}
+
+.blog .button {
+    margin-top: 5.625rem;
+    margin-bottom: 1.875rem;
+}
+
+.footer {
+    background-color: var(--background_color_two);
+}
+
+.footer_inner {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 4.375rem 0;
+}
```

Comment:
Overall no comments as this much resembles the example from the tutorial.

But there are no rules to support responsive design.